db/seeds.rb

```diff
@@ -5,20 +5,30 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-Card.create(name: "Copper", category: "Money", cost: 0, spending_power: 1, tags: "", desc: "", image: "copper.jpg")
-Card.create(name: "Silver", category: "Money", cost: 3, spending_power: 2, tags: "", desc: "", image: "silver.jpg")
-Card.create(name: "Gold", category: "Money", cost: 6, spending_power: 3, tags: "", desc: "", image: "gold.jpg")
-Card.create(name: "Estate", category: "Victory", cost: 2, victory_points: 1, tags: "", desc: "", image: "estate.jpg")
-Card.create(name: "Duchy", category: "Victory", cost: 5, victory_points: 3, tags: "", desc: "", image: "duchy.jpg")
-Card.create(name: "Province", category: "Victory", cost: 8, victory_points: 6, tags: "", desc: "", image: "province.jpg")
+if Rails.env.development?
+  image_url = "localhost:3000/card_images"
+elsif ENV[HEROKU_STAGING]
+  image_url = "https://tsdbcg.herokuapp.com/card_images"
+elsif Rails.env.production?
+  image_url = "https://tsdbcg.herokuapp.com/card_images"
+elsif Rails.env.test?
+  image_url = "localhost:3000/card_images"
+end
```

Comment:
Great! Glad we could get this logic sorted out. Variable has been added to Heroku already 👍